File: lib/java/android/src/main/java/alphaTab/alphaSkia/AlphaSkiaAndroid.java
package alphaTab.alphaSkia;

/**
 * The alphaSkia initializer for Android.
 */
public class AlphaSkiaAndroid extends AlphaSkiaPlatform {
    public static final AlphaSkiaAndroid INSTANCE = new AlphaSkiaAndroid();

    @Override
    public void initialize() {
        System.loadLibrary("alphaskiajni");
        setNativeLibLoaded();
    }
}

File: lib/kotlin/mpp/src/androidMain/kotlin/alphaSkia/AlphaSkiaAndroid.kt
package alphaTab.alphaSkia

/**
 * The alphaSkia initializer for Android.
 */
class AlphaSkiaAndroid : AlphaSkiaPlatform() {
    companion object {
        @JvmStatic
        val INSTANCE: AlphaSkiaAndroid = AlphaSkiaAndroid()
    }

    override fun initialize() {
        System.loadLibrary("alphaskiajni");
        setNativeLibLoaded();
    }
}

